### ISSUE TITLE
Add more Tensor-related examples

### ISSUE THREAD

**Issue by zaleslaw** [tensorflow/java-models#10]
- [ ] How to create Tensor and populate with data

- [ ] How to copy to/from tensor

- [ ] Cover basic operations on one or two tensors

- [ ] Export/import tensor to file

**Comment by gustavo-ren** [comment]
Hello @zaleslaw , I am learning about Tensorflow for Java and doing some codes with math operations between two tensors with more than one dimension, do you have interest in check them out?

**Comment by zaleslaw** [comment]
Of course, @gustavo-ren , feel free to create PR or post the link here with your examples

**Comment by gustavo-ren** [comment]
Hello @zaleslaw, here it is https://github.com/gustavo-ren/java-tensor-math/tree/master I implemented the four basic ops on array Tensors, feel free to tell me what you think.

**Comment by dhruvil410** [comment]
Hi @zaleslaw, in second task, copy to/from tensor means from Java array to tensor and from tensor to Java array?
